_includes/nav.html

```diff
@@ -14,7 +14,7 @@ <h1 class="alt-h4 lh-default v-align-middle">
       <nav class="text-bold text-right">
         <a class="text-inherit px-2 d-inline-block" href="/apps/">Apps</a>
         <a class="text-inherit px-2 d-inline-block" href="/docs/">Docs</a>
-        <a class="text-inherit px-2 d-inline-block" href="/contribute/">Community</a>
+        <a class="text-inherit px-2 d-inline-block" href="/community/">Community</a>
```

Comment:
We need to make sure this redirects properly: https://github.com/jekyll/jekyll-redirect-from

Reply:
Ah that's true. I will make it happen 👍